metricshub-engine/src/main/java/org/metricshub/engine/extension/ExtensionManager.java

```diff
@@ -276,23 +276,26 @@ public Optional<IConfiguration> buildConfigurationFromJsonNode(
 	}
 
 	/**
-	 * Aggregates connector stores from various provider extensions into a single {@link ConnectorStore}.
-	 * This method initializes a new {@link ConnectorStore} and iteratively loads data from each registered
+	 * Aggregates connector stores from various provider extensions into a single {@link RawConnectorStore}.
+	 * This method initializes a new {@link RawConnectorStore} and iteratively loads data from each registered
 	 * connector store provider extension. All connectors from each extension are added to the newly created store.
 	 *
 	 * This centralized store can be used to manage or interpret connectors across different extensions efficiently,
 	 * providing a unified view of all connectors available in MetricsHub.
 	 *
-	 * @return A {@link ConnectorStore} containing all connectors from various extensions combined into one store.
+	 * @return A {@link RawConnectorStore} containing all connectors from various extensions combined into one store.
 	 */
-	public ConnectorStore aggregateExtensionConnectorStores() {
-		final ConnectorStore connectorStore = new ConnectorStore();
-		connectorStore.setStore(new TreeMap<>(String.CASE_INSENSITIVE_ORDER));
+	public RawConnectorStore aggregateExtensionRawConnectorStores() {
+		final RawConnectorStore rawConnectorStore = new RawConnectorStore();
+		rawConnectorStore.setStore(new TreeMap<>(String.CASE_INSENSITIVE_ORDER));

```

Comment:
This shouldn't be performed because the store should be initialized with an empty TreeMap.